README.md

```diff
@@ -1,6 +1,64 @@
-# edgex-endorsement
+# EdgeX Endorsement
 
 ## Introduction
 
-Code, tools, and documentation for validating and endorsing device profiles written by third parties.
+Code, tools, and documentation for validating and endorsing device profiles written by third parties. At a high level you'll need to complete five steps to be eligible for the EdgeX Endorsement Program which will ensure that data provided meets all required components. 
+
+1) Deploy EdgeX using one of the provided cloud templates. Alternatively, you can leverage the latest released compose-file provided here: This ensures a vanilla version of EdgeX is being used without modification to ensure the highest compatibility.
```

Comment:
I would think you can't develop a device profile without having a running EdgeX instance, how would you deploy it and test it without having EdgeX running? Open to changing it, just seems backwards to me as it would be develop, deploy edgex, deploy, if it doesnt work, start back at step1 and repeat-- it just doesn't flow as easily to me.